engintron.sh

```diff
@@ -313,6 +313,18 @@ EOFS
 		systemctl enable nginx
 	fi
 
+  if [ ! -f /etc/chkserv.d/engintron ]; then
+    touch /etc/chkserv.d/engintron
+  fi
+
+  cat > "/etc/chkserv.d/engintron" <<EOF
+service[nginx]=x,x,x,service nginx restart,nginx,root
+EOF
+  if [ "$(grep -c 'engintron:1' /etc/chkserv.d/chkservd.conf)" -eq 0 ]; then
+    echo 'engintron:1' >> /etc/chkserv.d/chkservd.conf
+  fi
+  /scripts/restartsrv_chkservd
```

Comment:
Should also check if the following file exists and then create it, otherwise the WHM Services page will not list engintron

`echo "+" > /var/run/chkservd/engintron`

see my comment here, could create 2 checks for both port 80 and 443 https://github.com/engintron/engintron/issues/374#issuecomment-279142308